Not suitable for upstream

Reverting the default to Sans 11 and Monospace 11 instead of Cantarell 11
and Adobe Source Code Pro 10

https://bugzilla.redhat.com/show_bug.cgi?id=1715761

--- a/schemas/org.gnome.desktop.interface.gschema.xml.in	2020-01-15 15:02:17.198568711 -0800
+++ b/schemas/org.gnome.desktop.interface.gschema.xml.in	2020-01-15 15:03:24.060707908 -0800
@@ -149,7 +149,7 @@
       </description>
     </key>
     <key name="document-font-name" type="s">
-      <default>'Cantarell 11'</default>
+      <default>'Sans 11'</default>
       <summary>Document font</summary>
       <description>
         Name of the default font used for reading documents.
@@ -156,7 +156,7 @@
       </description>
     </key>
     <key name="monospace-font-name" type="s">
-      <default>'Source Code Pro 10'</default>
+      <default>'Monospace 11'</default>
       <summary>Monospace font</summary>
       <description>
         Name of a monospaced (fixed-width) font for use in locations like
